app/javascript/packs/application_css.scss

```diff
@@ -12,3 +12,39 @@
 footer.primary-footer {
   display: none;
 }
+
+#admin-bike-box img {
```

Comment:
I added I added an admin stylesheet in https://github.com/bikeindex/bike_index/pull/736 - merge conflict, plus the admin-bike-box should go there